packages/components/src/generic_modal/generic_modal.tsx

```diff
@@ -29,6 +29,7 @@ export type Props = {
     compassDesign?: boolean;
     backdrop?: boolean;
     backdropClassName?: string;
+    children: React.ReactNode;
```

Comment:
Added this as required, as a modal always has a body and it's used unconditionally in the render method.